src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -111,5 +108,52 @@ void getStations() {
     @DisplayName("지하철역을 제거한다.")
     @Test
     void deleteStation() {
+        // given
+        String stationName = "강남역";
+        String stationName2 = "역삼역";
+
+        ExtractableResponse<Response> response = 지하철역_생성(stationName);
+        response = 지하철역_생성(stationName2);
+        // when
+        지하철역_삭제(response.body().jsonPath().getInt("id"));
+
+        // then
+        List<String> stationNames = 지하철역_조회();
+        assertAll(
+                () -> assertThat(stationNames.size()).isEqualTo(1),
+                () -> assertThat(stationNames).containsAnyOf("강남역"),
+                () -> assertThat(stationNames).doesNotContain("역삼역")
+        );
```

Comment:
assertj 를 활용하신다면 메서드 체이닝을 통해 다음과 같이 검증을 하실 수 있어요 😆 
반영해주실 필요는 없고 참고만해주세요 🙇‍♂️
```suggestion
        assertThat(stationNames)
                .containsOnly("강남역")
                .doesNotContain("역삼역");
```

Reply:
감사합니다! 적용 해 보겠습니다!